samples/createHttpTask.js

```diff
@@ -18,17 +18,18 @@
 // sample-metadata:
 //   title: Cloud Tasks Create HTTP Target
 //   description: Create Cloud Tasks with a HTTP Target
+//   usage: node createHttpTask.js
```

Comment:
The idea here is to call out which arguments the user can pass to the program, so they know the usage info.  Can we make this something like:

```js
//    usage: node createHttpTask.js <projectId> <queueName> <location> <url> <payload> <delayInSeconds> 
```

Reply:
Now that I've add the default arguments, do you think this will confuse the user?